src/extension.ts

```diff
@@ -114,12 +135,17 @@ export function getFhirDocumentationName(document: TextDocument, position: Posit
     .replace(/\s/, ''); // If we matched the Extension special case, remove any spaces between Extension and :
 }
 
-export function getDocumentationUri(name: string): Uri {
+export function getDocumentationUri(name: string, version: string): Uri {
+  const versionPath = DOCUMENTATION_VERSION_PATHS.has(version)
+    ? DOCUMENTATION_VERSION_PATHS.get(version)
+    : '';
```

Comment:
I don't think there is any current situation that will result in `versionPath` becoming `''`, right?  Because the `completionProvider.fhirVersion` defaults to `4.0.1` if it can't figure out the version.

This means that unfound or invalid versions will go to `https://hl7.org/fhir/R4/${lowerName}.html`, _not_ `https://hl7.org/fhir/${lowerName}.html`.  I just want to ensure that was your intent -- because I see a test below that suggests otherwise.